tests/indicators/algorithms/relativeStrengthIndicatorSpec.js

```diff
@@ -77,14 +77,14 @@
 
             it('should calculate correctly with a windowSize of 2', function() {
                 rsi.windowSize(2);
-                expect(rsi(data)).toEqual([50, 50]);
+                expect(rsi(data)).toEqual([undefined, 50, 50]);
             });
 
             it('should calculate correctly with a windowSize of 3', function() {
                 rsi.windowSize(3);
-                expect(rsi(data)).toEqual([22.222222222222214]);
+                expect(rsi(data)).toEqual([undefined, undefined, 22.222222222222214]);
```

Comment:
I presume this is a side-effect of zip/mapping the two data series together, they have to be of the same length, so an algorithm must output an array of the same length as the input array?